Tasks/Pokulnicki/Task3/how_much_time.py

```diff
@@ -0,0 +1,116 @@
+import datetime as dt
+from datetime import datetime
+
+name_of_hours = {1 : "один", 2 : "два", 3 : "три",
+                 4 : "четыре", 5 : "пять", 6 : "шесть",
+                 7 : "семь", 8 : "восемь", 9 : "девять",
+                 10 : "десять", 11 : "одиннадцать", 
+                 12 : "двенадцать", 13 : "один"}
+
+name_of_hours_ordinal = {1 : "первого", 2 : "второго", 3 : "третьего",
+                         4 : "четвертого", 5 : "пятого", 6 : "шестого",
+                         7 : "седьмого", 8 : "восьмого", 9 : "девятого",
+                         10 : "десятого", 11 : "одиннадцатого", 12 : "двенадцатого",
+                         13 : "первого"}
+
+name_of_minute_upto30 = {1 : "одна", 2 : "две", 3 : "три",
+                         4 : "четыре", 5 : "пять", 6 : "шесть",
+                         7 : "семь", 8 : "восемь", 9 : "девять",
+                         10 : "десять", 11 : "одиннадцать", 12 : "двенадцать",
+                         13 : "тринадцать", 14 : "четырнадцать", 15 : "пятнадцать",
+                         16 : "шестнадцать", 17 : "семнадцать", 18 : "восемнадцать",
+                         19 : "девятнадцать", 20 : "двадцать", 21 : "двадцать одна",
+                         22 : "двадцать две", 23 : "двадцать три", 24 : "двадцать четыре",
+                         25 : "двадцать пять", 26 : "двадцать шесть", 27 : "двадцать семь",
+                         28 : "двадцать восемь", 29 : "двадцать девять"}
+
+name_of_minute_after30 = {31 : "тридцать одна", 32 : "тридцать две", 33 : "тридцать три",
+                          34 : "тридцать четыре", 35 : "тридцать пять", 36 : "тридцать шесть",
+                          37 : "тридцать семь", 38 : "тридцать восемь",39 : "тридцать девять",
+                          40 : "сорок", 41 : "сорок одна", 42 : "сорок две",
+                          43 : "сорок три", 44 : "сорок четыре"}
+
+name_of_minute_after45 = {1 : "одной", 2 : "двух", 3 : "трех",
+                          4 : "четырех", 5 : "пяти", 6 : "шести",
+                          7 : "семи", 8 : "восьми", 9 : "девяти",
+                          10 : "десяти", 11 : "одиннадцати", 12 : "двенадцати",
+                          13 : "тринадцати", 14 : "четырнадцати", 15 : "пятнадцати"}
```

Comment:
the data structure contains several duplicates, not the most optimal way to do this task